### ISSUE TITLE
Syntax highlighting contrast issues

### ISSUE THREAD

**Issue by mattgarrish** [w3c/respec#3849]
## Important info

- URL to affected spec: https://w3c.github.io/epub-specs/epub33/core/
- ReSpec version: 26.16.1
- [x] I did a "hard refresh", but it's still busted.

## Description of problem

In getting our accessibility horizontal review, it was pointed out that the syntax highlighting violates WCAG contrast requirements. Looking at the spec source, it looks like there are conflicting background rules that prevent a lighter grey background colour from being applied to `code` elements.

The `style` element with the id `respec-mainstyle` includes this rule:

```
.hljs{background:0 0!important}
```

which overrides the following background colour setting in the `script` element that is injected with the hljs styles:

```
.hljs{display:block;overflow-x:auto;padding:.5em;color:#383a42;background:#fafafa}
```

The result is the background colour for the `example` class gets applied (`rgb(252,250,238)`) and that causes the contrast issues. Is this working as designed and I'll need to manually override, or can the first declaration be removed?

**Comment by marcoscaceres** [comment]
yeah, that doesn't seem right... I'll need to track down if the hljs template is doing that.

**Comment by sanderke** [comment]
According to this comment, overriding the code block background seems intentional:
https://github.com/w3c/respec/blob/ffc4b52c4500fa2a12a271ed6366358a5b1fa55c/src/styles/respec.css.js#L20-L23
The colour of the example boxes (`#fcfaee`) is darkened for `<pre>` elements:
https://github.com/w3c/respec/blob/ffc4b52c4500fa2a12a271ed6366358a5b1fa55c/src/styles/examples.css.js#L41-L43
The resulting colour (`#f4f2e7`) does not provide sufficient contrast with highlighted text (e.g. `#ca4706`). The overruled background colour (`#fafafa`) would have passed WCAG AA.
